internal/scripts/scripts.go

```diff
@@ -33,6 +37,8 @@ type ContentValue struct {
 	RootDir    string
 	CheckRead  func(path string) error
 	CheckWrite func(path string) error
+	// Create should be called to write to file.
+	Create     func(opts *fsutil.CreateOptions) error
```

Comment:
I am still not fond of the naming. We'll have to think about it some more.

Reply:
A few ideas:

```
WriteFile
Write
CreateFile
```

I like `WriteFile`.

Reply:
Let's make it `CreateFile` and postpone this discussion to the meeting with Gustavo.

Reply:
Done.